Assets/_Laz/Tests/PatrolBehaviourTests.cs

```diff
@@ -55,15 +55,15 @@ public IEnumerator Test_PatrolMovement_Reset()
         {
             for (int i = 0; i < 5; i++)
             {
-                yield return new WaitForEndOfFrame();
+                yield return new WaitForFixedUpdate();
             }
 
             var patrollingObject = GameObject.FindObjectOfType<PatrolBehaviour>();
             patrollingObject.gameObject.transform.position = Vector3.left;
             patrollingObject.Initialize();
             patrollingObject.Reset();
                 
-            yield return new WaitForEndOfFrame();
+            yield return new WaitForSeconds(0);
```

Comment:
Is this the work around?

Reply:
yeah if I used WaitForFixedUpdate it failed so this was trial and error